api-inference-community/docker_images/generic/requirements.txt

```diff
@@ -1,2 +1,8 @@
 starlette==0.14.2
-api-inference-community==0.0.8
+api-inference-community==0.0.13
+torch>=1.7.0
+scikit-learn>=0.23.2
+scipy>=1.5.4
+numba>=0.48
```

Comment:
out of curiosity, why do we need `numba` in the base requirements?

Reply:
Good question, I removed it. I still need to decide which will be the pre-installed requirements and their respective versions, so this is mostly a temporal placeholder.